includes/class-wcpdf-admin.php

```diff
@@ -56,9 +56,15 @@ public function review_plugin_notice() {
 		if ( get_option( 'wpo_wcpdf_review_notice_dismissed' ) !== false ) {
 			return;
 		} else {
-			if ( isset( $_GET['wpo_wcpdf_dismis_review'] ) ) {
-				update_option( 'wpo_wcpdf_review_notice_dismissed', true );
-				return;
+			if ( isset( $_REQUEST['wpo_wcpdf_dismis_review'] ) && isset( $_REQUEST['_wpnonce'] ) ) {
```

Comment:
Is not your fault because it was like this before, but `dismiss` has two `s`

Reply:
I'm not sure if the change I made is correct. Please correct me if it is not correct.